Code-4-Community/speak-for-the-trees-backend-v2
# Clot.reject image

## Why

Resolves #219 

This allows the admins to send rejection email to the users who uploaded inappropriate images. 

## This PR
We added a new email template "RejectImageEmail.html" to be used for image rejection email. 

## Verification Steps

Note that it will only send email when a rejection reason is given. When a reason is not given, it will delete the user image in database by using deleteSiteImage without sending the email.
